mmdet/models/backbones/hrnet.py

```diff
@@ -0,0 +1,517 @@
+import logging
+
+import torch.nn as nn
+from mmcv.cnn import constant_init, kaiming_init
+from mmcv.runner import load_checkpoint
+
+from ..registry import BACKBONES
+from ..utils import build_norm_layer, build_conv_layer
+from .resnet import BasicBlock, Bottleneck
+
+
+class HRModule(nn.Module):
+
```

Comment:
No blank line is needed between the class definition and docstring and a blank line is needed between the docstring and init method.